contrib/labstack/echo/example_test.go

```diff
@@ -0,0 +1,37 @@
+package echo
+
+import (
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+
+	"github.com/labstack/echo"
+)
+
+// To start tracing requests, add the trace middleware to your echo router.
+func Example() {
+	// Create a echo.Enechoe
+	r := echo.New()
+
+	// Use the tracer middleware with your desired service name.
+	r.Use(Middleware("my-web-app"))
+
+	// Set up some endpoints.
+	r.GET("/hello", func(c echo.Context) error {
+		return c.String(200, "hello world!")
+	})
+
+	// And start gathering request traces.
+	r.Start(":8080")
+}
+
+func Example_spanFromContext() {
+	r := echo.New()
+	r.Use(Middleware("image-encoder"))
+	r.GET("/image/encode", func(c echo.Context) error {
+		// create a child span to track operation timing.
+		encodeSpan, _ := tracer.StartSpanFromContext(c.Request().Context(), "image.encode")
+		// encode a image
```

Comment:
Let's just use:
```go

// do something ...

```
(notice the blank lines before and after

Reply:
i got it. thank you.

Reply:
You didn't apply this comment.